### ISSUE TITLE
Terms to describe EOV, EBV, and indicators

### ISSUE THREAD

**Issue by kmexter** [marco-bolo/dataset-catalogue#15]
Related to issue#5

In the dataset-gather googlesheet we have 2 columns in which people should indicate the nature of the EOV/EBV and indicator data that their dataset concerns (where those data _are_ those parameters, of course)
This is in the coverage tab: https://docs.google.com/spreadsheets/d/1Wmjpvj4FI8RK926RU3XsM8yDyKGG53PLZFhfiugIRSk/edit#gid=52011750
For the meanwhile, we have agreed to let this be a free-text entry and we will meanwhile look at we get here and come up with a controlled set of terms to use. This issue is to track our progress on this work

**Comment by pieterprovoost** [comment]
If possible I would like to at least provide some controlled examples for the GOOS EOVs, but still need to look into options (one was adding them to ENVO).

**Comment by kmexter** [comment]
This is something we should discuss in WP1, but as we have very little information at present on the EVs and indicators we are expecting, we cannot move on this for now. However, it would be handy if those who know these things could list in here the specific names and their URLs that do exist for EVs and indicators. I know that Pieter and PLB should have this info, as well as  I think Paolo and Dan, so I have assigned this issue to you guys to get your input.

**Comment by pbuttigieg** [comment]
xref https://github.com/EnvironmentOntology/envo/pull/1568

I've completed the BioEco EOV classes. The IRIs / PIDs are not released yet, but will not change. 

![image](https://github.com/user-attachments/assets/bb51ad88-6252-4882-a2e0-0d4223b3240e)

**Comment by kmexter** [comment]
cool, so we should tell the WPs about this at the GA so they can make theirs to these

**Comment by pbuttigieg** [comment]
Yes, and further discussion with the Panel has shown we need unpacking further to deal with richness, evenness, alpha/beta/gamma diversity, and composition. There will be classes for each of those, which should allow our WPs to be granular with their data markup

**Comment by kmexter** [comment]
We are updating the googlesheets for the EOVs (see https://github.com/marco-bolo/dataset-catalogue/issues/36 for a summary of changes) but still need to discuss how to collect
- indicators
- EBVs

In addition, we will need to discuss how to include these in the json-ld files: specifically, do we want to just have them as a measurement (which will be described via a string and URI, I assume) or do we want to very specifically flag them as EOV, EBV, Indicator so that is clear from the field itself, rather than only after someone has gone to the URL of the parameter or read the string?

I mean this: measurements are included as variableMeasured. You give the string and URL as properties. Do we _also_ want to add a property that says "EOV" or "EBV"?
